server/middleware/configureEstablishment.js

```diff
@@ -19,18 +18,22 @@ const configureEstablishment = () => (req, res, next) => {
       .split('.')[0]
       .replace(replaceUrl, '');
 
+    const establishmentId = getEstablishmentId(establishmentName);
+
     req.session.id = uuid();
-    req.session.establishmentName = establishmentName;
-    req.session.establishmentId = getEstablishmentId(establishmentName);
-    req.session.establishmentPersonalisationEnabled = getEstablishmentPersonalisation(
-      req.session.establishmentId,
-    );
+    if (typeof establishmentId === 'undefined') {
```

Comment:
we now only set establishment details if we can link to a real establishment.
Previously we would set the establishment name to `localhost:3000` as the regex is a little bit dodgy